translations/ssofact.de.po

```diff
@@ -0,0 +1,177 @@
+#  modules/contrib/ssofact/config/schema/ssofact.schema.yml: n/a
+#  modules/contrib/ssofact/src/Plugin/OpenIDConnectClient/Ssofact.php: n/a
+#  modules/contrib/ssofact/src/Form/SsofactLoginForm.php: n/a
+#  modules/contrib/ssofact/src/Plugin/Block/SsofactRegisterBlock.php: n/a
+#  modules/contrib/ssofact/src/Plugin/Block/SsofactServerRegisterBlock.php: n/a
+#  modules/contrib/ssofact/src/Form/SsofactRegisterForm.php: n/a
+#  modules/contrib/ssofact/src/Form/SsofactServerRegisterForm.php: n/a
+#  modules/contrib/ssofact/src/Plugin/Block/SsofactLoginBlock.php: n/a
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: PROJECT VERSION\n"
+"POT-Creation-Date: 2018-07-29 19:49+0200\n"
+"PO-Revision-Date: YYYY-mm-DD HH:MM+ZZZZ\n"
+"Last-Translator: NAME <EMAIL@ADDRESS>\n"
+"Language-Team: LANGUAGE <EMAIL@ADDRESS>\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=INTEGER; plural=EXPRESSION;\n"
+
+#: modules/contrib/ssofact/ssofact.module:23
+msgid "About"
+msgstr "Ãœber das Modul"
+
+#: modules/contrib/ssofact/ssofact.module:24 modules/contrib/ssofact/ssofact.info.yml:0
+msgid "OpenID Connect provider for Newsfactory ssoFACT."
+msgstr "OpenID Connect Anbieter fÃ¼r Newsfactory ssoFACT"
+
+#: modules/contrib/ssofact/ssofact.info.yml:0
+msgid "OpenID Connect Newsfactory ssoFACT Client"
+msgstr "OpenID Connect Newsfactory ssoFACT Client"
+
+#: modules/contrib/ssofact/ssofact.info.yml:0
+msgid "Other"
+msgstr "Andere"
+
+#: modules/contrib/ssofact/ssofact.routing.yml:0
+msgid "OpenID Connect ssoFACT login form redirect page"
+msgstr "OpenID Connect ssoFACT Redirect-Seite fÃ¼r Login-Formular"
+
+#: modules/contrib/ssofact/ssofact.routing.yml:0
+msgid "ssoFACT"
+msgstr "ssoFACT"
+
+#: modules/contrib/ssofact/config/schema/ssofact.schema.yml:0
+msgid "OpenID Connect Newsfactory ssoFACT settings"
+msgstr "OpenID Connect Newsfactory ssoFACT Einstellungen"
+
+#: modules/contrib/ssofact/config/schema/ssofact.schema.yml:0
+msgid "Enable client"
+msgstr "Client aktivieren"
+
+#: modules/contrib/ssofact/config/schema/ssofact.schema.yml:0
+msgid "Client ID"
+msgstr Client-ID"
+
+#: modules/contrib/ssofact/config/schema/ssofact.schema.yml:0
+msgid "Client secret"
+msgstr "Client Secret"
+
+#: modules/contrib/ssofact/config/schema/ssofact.schema.yml:0 modules/contrib/ssofact/src/Plugin/OpenIDConnectClient/Ssofact.php:50
+msgid "Server domain"
+msgstr "Server-Domain"
+
+#: modules/contrib/ssofact/config/schema/ssofact.schema.yml:0 modules/contrib/ssofact/src/Plugin/OpenIDConnectClient/Ssofact.php:58
+msgid "Scope"
+msgstr "Umfang"
+
+#: modules/contrib/ssofact/src/Form/SsofactLoginForm.php:114
+msgid "Email address"
+msgstr "E-Mailadresse"
+
+#: modules/contrib/ssofact/src/Form/SsofactLoginForm.php:129
+msgid "Password"
+msgstr "Passwort"
+
+#: modules/contrib/ssofact/src/Form/SsofactLoginForm.php:137
+msgid "Stay logged-in"
+msgstr "Eingeloggt bleiben"
```

Comment:
Needs to be the same string as in https://stage-login.stimme.de/ — "Angemeldet bleiben"